lib/mayaUsd/render/MaterialXGenOgsXml/GlslOcioNodeImpl.cpp

```diff
@@ -62,8 +62,20 @@ constexpr const char ATTR_NAME[] = "name";
 constexpr const char ATTR_LANGUAGE[] = "language";
 constexpr const char ATTR_VAL[] = "val";
 
-std::map<std::string, pugi::xml_document> knownOCIOFragments;
-std::vector<std::string>                  knownOCIOImplementations;
+struct OcioData
+{
+    std::string              fragName;
+    std::string              colorInput;
+    std::string              colorOutput;
+    std::vector<std::string> extraParams;
+    std::vector<std::string> samplerNames;
+    std::string              functionName;
+    std::string              sourceCode;
+};
+
+std::map<std::string, OcioData> knownOCIOFragments;
+std::vector<std::string>        knownOCIOImplementations;
+DocumentPtr                     knownLibrary;
```

Comment:
This node now manages the list of known OCIO NodeDefs. Moved from material.cpp to keep the info centralized.